nix/modules/devShells.nix

```diff
@@ -65,6 +65,7 @@
               export CARGO_CACHE_RUSTC_INFO=1
               export PATH="$CARGO_INSTALL_ROOT/bin:$PATH"
               export NIX_PATH="nixpkgs=${pkgs.path}"
+              ${config.pre-commit.installationScript}
```

Comment:
the holonix devshell inherits from rustDev. i don't know if it's appropriate to make this the default for all holonix users so i suggest we move this to coreDev.

Reply:
This makes the pre-commit checks become part of `nix flake check`, see: https://flake.parts/options/pre-commit-hooks-nix.html#opt-perSystem.pre-commit.check.enable